github-pr

```diff
@@ -12,11 +12,19 @@ def _check_required_fields(required, **args):
             sys.exit(1)
 
 
+def _return_status(commits):
+    commit_statuses = []
+    for commit in commits:
+        for commit_status in commit.get_statuses():
+            commit_statuses.append(commit_status.state)
+    return commit_statuses[-1]
```

Comment:
when you say "look up the most recent status for the most recent commit" im not sure how to go about doing that other than building the list that i am doing on line 19. i can reference the last commit in the object that is returned from get_commits().